config/samples/calico-cni.yaml

```diff
@@ -4,12 +4,18 @@ metadata:
   name: calico-cni
 spec:
   clusterSelector:
-    matchLabels:
-      calicoCNI: enabled
+  # Target workload clusters with specific labels.
+  #  matchLabels:
+  #    calicoCNI: enabled
+  # Target all workload clusters.
+    matchExpressions:
```

Comment:
I see. I'll open the issue.
Since the goal is to target all workload clusters, should I change it to the following instead of using `matchExpressions`?
(This one is much simpler)
```
  # Target all workload clusters.
    matchLabels: {}
```

Reply:
I created the issue #15.

Reply:
Thanks for opening the issue! And let me double check about that -- I'm not certain if an empty label selector would select all clusters or no clusters.

Reply:
I have confirmed that if the "HelmChartProxy" matchlabels setting is set to empty, the matching is unconditional.
Since it is a rather common Kubernetes setting to set the label selector to an empty setting for unconditional matching, I'd like to change the setting.

Reply:
Sure, I think that seems reasonable. Just to clarify, you're suggesting that empty labels should select everything like a wildcard or * sign?

Reply:
That is correct.

Reply:
Also, the URL for calico helm repo seems to have changed, so I updated the YAML.
(I intended to fix it in https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/pull/14/commits/a222e9c36cce68a866afae80abcd7b42b033fbc0)
- https://docs.tigera.io/calico/3.25/getting-started/kubernetes/helm#download-the-helm-chart